src/app/thankYou/thankYou.tpl.html

```diff
@@ -125,7 +125,7 @@ <h4 class="panel-name" translate>Gift Summary</h4>
                               <td class="td-frequency">
                                 <span class="giftsum-about giftsum-title">{{item.rate.display}}</span>
                                 <span class="giftsum-frequency giftsum-title">
-                                  <span ng-if="item.rate.recurrence.interval !== 'NA'" translate>First Gift: XX/XX/XXXX</span>
+                                  <span ng-if="item.rate.recurrence.interval !== 'NA'" translate>First Gift: {{item.rate['start-date'].value | date:'MMM d, yyyy':'UTC'}}</span>
```

Comment:
@OzzieOrca I'm a little confused on this?  The dates should all be in the local timezone, so won't this add or remove hours to the correct date?